app/scripts/modules/core/projects/dashboard/pipeline/ProjectPipeline.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
Discussed in person; JSX transpiles down to `React.createElement()` so we need `React`. This could be changed to `import React, { Component } from 'react'` if you add custom typescript config. Feels even dirtier. Agreed to leave it as is.